code/modules/mob/mob.dm

```diff
@@ -1006,6 +1006,7 @@
 
 /mob/physically_destroyed(var/skip_qdel)
 	SHOULD_CALL_PARENT(FALSE)
+	dump_contents()
```

Comment:
That probably breaks things being thrown around in human/gib(). There's a loop to throw around the mob's contents. So instead of throwing the contents its just going to drop it down.

Reply:
Mob gear is stripped and thrown before parent is called so nope it works fine.